src/main/java/htsjdk/variant/vcf/VCFUtils.java

```diff
@@ -250,6 +254,32 @@ else if (vcfFile.getAbsolutePath().endsWith(IOUtil.COMPRESSED_VCF_FILE_EXTENSION
         return output;
     }
 
+    /**
+     * Parses a String as a Double, being tolerant for lower-case NaN (nan).
+     */
+    public static double parseDoubleAccordingToVcfSpec(final String str) {
+        try {
+            return Double.parseDouble(str);
+        } catch (NumberFormatException e) {
+            final Matcher matcher = infOrNanPattern.matcher(str);
+            if (matcher.matches()) {
+                final double ret;
+                if (matcher.group("inf") == null) {
+                    ret = Double.NaN;
+                } else {
+                    if (matcher.group("sign").equals("-")) {
+                        ret = Double.NEGATIVE_INFINITY;
```

Comment:
The test below appears to test this case but it's showing up as not covered. Are you sure that this code path is being tested by the code below?